### PR TITLE
[Dashboard] Remove <Center />

### PR DESCRIPTION
**Pull request by kien-ngo**
## Problem solved

Short description of the bug fixed or feature added

---

## PR-Codex overview
This PR focuses on replacing instances of the `Center` component from `@chakra-ui/react` with `div` elements styled with Flexbox for better layout control across various components in the application.

### Detailed summary
- Replaced `Center` component with `div` and Flexbox styles in multiple files for layout adjustments.
- Updated loading indicators to use `div` elements instead of `Center`.
- Adjusted alignment and spacing in components for improved responsiveness.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by kien-ngo** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/4765?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#4765** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/4765?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#4750** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/4750?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4749** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/4749?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4748** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/4748?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @kien-ngo and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/dashboard/src/components/FTUX/components/GraphicContainer.tsx [hunk 1]
```diff
@@ -22,6 +22,7 @@ export const GraphicContainer: React.FC<SlideStateProps> = ({
 
   return (
     <div className="w-full relative aspect-[16/9] md:aspect-square">
+      {/* @to be completed */}
       <Center
```
Comment (kien-ngo) [inline apps/dashboard/src/components/FTUX/components/GraphicContainer.tsx:26]: @MananTank I might need help with this one ^

File: apps/dashboard/src/components/buttons/TransactionButton.tsx [hunk 2]
```diff
@@ -119,6 +119,7 @@ export const TransactionButton: React.FC<TransactionButtonProps> = ({
                 : `This action will trigger ${transactionCount} ${transactionCount > 1 ? "transactions" : "transaction"}`
             }
           >
+            {/* to be completed */}
```
Comment (kien-ngo) [inline apps/dashboard/src/components/buttons/TransactionButton.tsx:122]: also might need help with this

Comment (MananTank) [reply apps/dashboard/src/components/buttons/TransactionButton.tsx:122]: Let's just ignore this for now, We'll remove this when refactoring TransactionButton

File: apps/dashboard/src/components/product-pages/common/GameCard.tsx [hunk 3]
```diff
@@ -70,7 +70,7 @@ export const GameCard: React.FC<GameCardProps> = ({
               (max-width: 1200px) 50vw,
               33vw"
         />
-        <Center position="absolute" top={0} bottom={0} left={0} right={0}>
+        <div className="flex items-center justify-center absolute top-0 bottom-0 left-0 right-0">
```
Comment (MananTank) [inline apps/dashboard/src/components/product-pages/common/GameCard.tsx:73]: you can use `inset-0` instead to set all 4 positions

File: apps/dashboard/src/components/shared/TWQueryTable.tsx [hunk 4]
```diff
@@ -265,15 +264,9 @@ function Pagination<TInputData>(paginationProps: PaginationProps<TInputData>) {
...
                         {buttonStringTemplate}
                       </Box>
-                      <Center
-                        position="absolute"
-                        top={0}
-                        right={0}
-                        left={0}
-                        bottom={0}
-                      >
+                      <div className="flex items-center justify-center absolute top-0 right-0 left-0 bottom-0">
```
Comment (MananTank) [inline apps/dashboard/src/components/shared/TWQueryTable.tsx:267]: use `inset-0`

File: apps/dashboard/src/core-ui/batch-upload/upload-step.tsx [hunk 5]
```diff
@@ -31,9 +31,13 @@ export const UploadStep: React.FC<UploadStepProps> = ({
...
           <div className="aspect-square w-full md:w-1/2">
-            <Center
-              borderRadius="md"
+            <div
+              className={cn(
+                "rounded-md cursor-pointer border border-border h-full hover:border-blue-500",
+                hasFailed ? "bg-red-200" : "bg-card",
+              )}
               {...getRootProps()}
+<<<<<<< HEAD
```
Comment (MananTank) [inline apps/dashboard/src/core-ui/batch-upload/upload-step.tsx:40]: Merge conflict here

**Review by MananTank — CHANGES_REQUESTED** [review]
remove the commit merge conflict markers and address other nits

File: apps/dashboard/src/components/product-pages/common/GameCard.tsx (hunk 3 above)
Comment (jnsdls) [reply apps/dashboard/src/components/product-pages/common/GameCard.tsx]: til tbh

Comment (kien-ngo) [reply apps/dashboard/src/components/product-pages/common/GameCard.tsx]: nicee

**Review by MananTank — APPROVED** [review]

**Pull request merged**